openclaw_hooks/nima-recall-live/lazy_recall_v3.py

```
@@ -0,0 +1,432 @@
+# =============================================================================
+
+VOYAGE_API_KEY = "pa-Key1T8F11CHGsd6lysFnTo9b-xzCdhHiauzzTaa6XMA"
+
+def get_embedding_voyage(text: str) -> Optional[np.ndarray]:
+    """Get embedding from Voyage API."""
+    try:
+        import voyageai
+        client = voyageai.Client(api_key=VOYAGE_API_KEY)
+        result = client.embed([text[:500]], model="voyage-3-lite")
+        return np.array(result.embeddings[0], dtype=np.float32)
+    except Exception as e:
```

Ruff (0.15.0)

[warning] Do not catch blind exception: `Exception`

(BLE001)